osc-ui/src/main/java/org/osc/core/broker/view/securityinterface/SecurityGroupInterfaceSubView.java

```diff
@@ -162,10 +162,10 @@ public Object generateCell(CustomTable source, Object itemId, Object columnId) {
             public Object generateCell(CustomTable source, Object itemId, Object columnId) {
                 SecurityGroupInterfaceDto dto = SecurityGroupInterfaceSubView.this.tableContainer.getItem(itemId)
                         .getBean();
-                if (dto.getPolicyName() == null) {
+                if (dto.getPolicies().iterator().next().getPolicyName() == null) {
```

Comment:
policies could be empty depending on the support policy mapping flag....we cannot assume a iterator or an element here..please handle